### DISCUSSION TITLE
photoprism library to mounted network share

### DISCUSSION THREAD

**Discussion by datadrivenlucas** [photoprism/photoprism#4241] [Q&A]
Hi,

I would like photoprism to use the photos in a certain location on a network folder. So I only have to update that folder to let photoprism show more pictures.

I mounted my network share "trial" in the host server. See below

![image](https://github.com/photoprism/photoprism/assets/17165755/861e13ff-d403-4be3-a95f-d1d04ba31053)

I thought adding this to the YAML file would let photoprism see my mount using[ the official documentation.](https://docs.photoprism.app/getting-started/advanced/docker-volumes/)

```
volumes: 
      - "/mnt/trial/Pictures:/photoprism/originals/media" 
```

But somehow it doesn't. Can someone point me in the right direction? 

Edit:I pushed some buttons in the interface and suddenly it started indexing a newly added folder. I think it's working.
Edit2: You have to scan your library in the interface of photoprism when you have added photos to the mounted drive.

**Comment by graciousgrey** [comment]
Glad you got it working :)

You can find more information on network storage and auto- indexing here:
- https://docs.photoprism.app/getting-started/troubleshooting/docker/#network-storage
- https://docs.photoprism.app/user-guide/library/originals/#automatic-indexing
